examples/lambda-functions/get-by-id.ts

```diff
@@ -30,6 +30,14 @@ const tableName = process.env.SAMPLE_TABLE;
 
 class Lambda implements LambdaInterface {
 
+  @tracer.captureMethod()
+  public async getUuid(): Promise<string> {
+    // Request a sample random uuid from a webservice
+    const res = await got('https://httpbin.org/uuid');
+    
+    return JSON.parse(res.body).uuid;
+  }
+
   @tracer.captureLambdaHandler()
```

Comment:
I would pass the `captureResponse: false` option.

In this example it might not be an issue, but it's a good chance to show (& explain in a comment) that normally the decorator would add the response as metadata on the segment, but if the returned object is bigger than 64kb this will hit the X-Ray segment size limit.

Same applies to the other two functions, middleware has the same option.